### ISSUE TITLE
Gateway - Failed to connect using existed wallet

### ISSUE THREAD

**Issue by nikspz** [hummingbot/gateway#74]
Gateway - Failed to connect to the same chain using existed wallet. Issue reproduced using Avalanche and ethereum, polygon, BSC chains

**Steps To Reproduce**
1. clone and install latest development branch
2. gateway connect one connector with `one chain` (for example pangolin avalanche)
3. check balance to make sure its connected
4. gateway connect connector with the `same chain` (for example traderjoe avalanche)
5. Do you want to connect to avalanche-avalanche with one of your existing wallets on Gateway? (Yes/No) >>> Yes  
6. Pick the wallet if its showed

Actual:
second connector `not connected when chose case to connect existed wallet`

Expected:
Bot able to connect using existed wallet

```
2023-03-27 07:33:13,077 - 2588670 - hummingbot.core.utils.async_utils - ERROR - Unhandled error in background task: local variable 'additional_prompt_values' referenced before assignment
Traceback (most recent call last):
  File "/home/ubuntu/development/hummingbot/core/utils/async_utils.py", line 9, in safe_wrapper
    return await c
  File "/home/ubuntu/development/hummingbot/client/command/gateway_command.py", line 315, in _gateway_connect
    additional_prompt_values=additional_prompt_values,
UnboundLocalError: local variable 'additional_prompt_values' referenced before assignment
```

Logs:
[logs_hummingbot.log](https://github.com/hummingbot/gateway/files/11076005/logs_hummingbot.log)

Loom video:
https://www.loom.com/share/ac7bdd468f484454bbc2dbe98ed36d3a

![image](https://user-images.githubusercontent.com/83953535/227871936-c405969f-12c0-44e4-88c3-8d49fbcac607.png)

**Release version**
dev-1.14.0 (commit 65e38b57f7626215f22e8b4139cb8df0bcf6a812)

NOTE: User still able to connect entering his private key, issue only in connecting using existed wallet

**Comment by nikspz** [comment]
Severity: P2
Bounty: 50,000 (https://hummingbot.org/maintenance/bugs/)
Rationale: An issue related to connect gateway connectors (Connection between Client and Exchange)

**Comment by klpanagi** [comment]
I would like this bounty to be assigned to me.

**Comment by nikspz** [comment]
Assigning this issue to @klpanagi . Thank you for your contribution!

**Comment by klpanagi** [comment]
@nikspz This issue is a result of a bug in the hummingbot/hummingbot codebase.

I am currently working on a fix. I will update this issue for convenience

**Comment by klpanagi** [comment]
Here is the PR that resolves this issue: https://github.com/hummingbot/hummingbot/pull/6178

**Comment by nikspz** [comment]
Fix provided here: hummingbot/hummingbot#6178, pending payment

**Comment by nikspz** [comment]
Closing issue as paid
